### ISSUE TITLE
Category, Functor, Monad, Comonad, etc. should mention algebraic laws

### ISSUE THREAD

**Issue by MostAwesomeDude** [hemanth/functional-programming-jargon#215]
For example, the algebraic laws for categories say that, for any arrow `f`. `f;id == id;f == f`. These laws are essential for functors and monads.

**Comment by raegen** [comment]
I consider myself somewhat literate when it comes to functional programming (it was love at first sight), so naturally (or not?), I went straight down to the monad definition, kind of hoping for a mindblowing simple (possibly wrong, but practical) description. Next I went to Issues to create an issue to show my disappointment but thinking I ought to keep it constructive. Imagine my dismay and complacency when I realized somebody has already done it.
tldr. It's not an understatement if I say I do not understand the aforementioned definitions.
